autonomi/src/client/archive.rs

```diff
@@ -160,14 +165,24 @@ impl Client {
         let bytes = archive
             .into_bytes()
             .map_err(|e| PutError::Serialization(format!("Failed to serialize archive: {e:?}")))?;
-        self.data_put(bytes, wallet.into()).await
+        let result = self.data_put(bytes, wallet.into()).await;
+        debug!(
+            "Uploaded archive {:?} to the network and the address is {:?}",
+            archive, result
+        );
```

Comment:
```suggestion
        debug!("Uploaded archive {archive:?} to the network and the address is {result:?}");
```

Tiny nitpick, but you can use the variable directly inside the `{}` thingy.

Reply:
Could do the same everywhere!